### PR TITLE
Tourniquet duplicity and stringtable fixes

### PR DESCRIPTION
**Pull request by YetheSamartaka**
- removes settable tourniquet time option since it is added by ACE

### CODE REVIEW

**Comment by Conrimaceogain** [comment]
LGTM, let's make sure that things play happily with the ACE settings and then we should be good to merge.

**Comment by YetheSamartaka** [comment]
Indeed

**Review by Conrimaceogain — APPROVED** [review]

File: addons/breathing/stringtable.xml [hunk 1]
```diff
@@ -153,8 +153,8 @@
             <Chinese>心率監測器: 心率: %1 血氧量: %2</Chinese>
             <Czech>Pulzní oxymetr: HR: %1 Sp02: %2</Czech>
             <Korean>맥박 산소포화도: HR: %1 Sp02: %2</Korean>
-            <French>Oxymètre de pouls: HR: 1% SpO2: 2%</French>
-            <Turkish>Nabız Oksimetresi: HR:% 1 Sp02:% 2</Turkish>
+            <French>Oxymètre de pouls: HR: %1 SpO2: %2</French>
+            <Turkish>Nabız Oksimetresi: HR:%1 Sp02:%2</Turkish>
```
Comment (Tomcat-SG) [inline addons/breathing/stringtable.xml]:
```suggestion
            <Turkish>Nabız Oksimetresi: HR: %1 Sp02: %2</Turkish>
```

File: addons/breathing/stringtable.xml [hunk 2]
```diff
@@ -215,9 +215,9 @@
 				%1SpO2 au delà 70% -&gt; Oui
 				%1SpO2 sous 70% -&gt; Non -&gt; ass. Respiration</French>
             <Turkish>Nefes
-				% 1 yeterince havalandırıldı mı?
-				% 1Sp02% 70'in üzerinde -> Evet
-				% 1Sp02% 70'in altında -> Hayır -> eşek. Nefes</Turkish>
+				%1yeterince havalandırıldı mı?
+				%1Sp02% 70'in üzerinde -> Evet
```
Comment (Tomcat-SG) [inline addons/breathing/stringtable.xml]:
```suggestion
				%1Sp02 70% 'in üzerinde -> Evet
```

Comment (Tomcat-SG) [inline addons/breathing/stringtable.xml]:
```suggestion
				%1Sp02 70% 'in üzerinde -> Evet
```

File: addons/breathing/stringtable.xml [hunk 3]
```diff
@@ -215,9 +215,9 @@
...
+				%1Sp02% 70'in altında -> Hayır -> eşek. Nefes</Turkish>
```
Comment (Tomcat-SG) [inline addons/breathing/stringtable.xml]:
```suggestion
				%1Sp02 70% 'in altında -> Hayır -> eşek. Nefes</Turkish>
```

**Review by Tomcat-SG — CHANGES_REQUESTED** [review]
Pls fix these things. Im not sure if the % on the turkish translation could cause at this position an error, because it is followed by a number, but it is better in this way, so it cant cause any errors

**Review by Tomcat-SG — APPROVED** [review]

**Pull request merged**